docs/prepping-local-system-runbook-linux.md

```diff
@@ -75,11 +75,11 @@ Helm provides a script that you can download and then run to install the latest
 1. To download the Helm installation script from their Git repository, from
```

Comment:
To download the Helm installation script from the Pega Github repository to your home directory on your local system, enter: